### ISSUE TITLE
metrics reports single master_link_status and master_sync_in_progress with keyDB multi-master

### ISSUE THREAD

**Issue by jtorkkel** [oliver006/redis_exporter#371]
KeyDB support Multimaster feature where more than two active master exists and redis exported should support multiple masters correctly.

Current behavior:
Redis-exporter expose only "first connected" master  "master_link_status" and "master_sync_in_progress" metrics. 

Expected behavior:
Redis-exporter should expose the metrics for all connected masters with label indicating master index, and maybe aggregated status and sync_in_progress
Also other instance based info like master_host and master_port should be exposed.

Below Info listing after KeyDB instance 127.0.0.1:8881 has been started and replication has completed with 8882 and 8884, but not with 8883.
Prometheus exported will report the status of first index 0 == :8881 indicating status=up and sync not in progress.

Tested version 1.5.2

Debug info:
```
127.0.0.1:8881> info replication
# Replication
role:active-replica
Master 0:
master_host:127.0.0.1
master_port:8882
master_link_status:up
master_last_io_seconds_ago:0
master_sync_in_progress:0
slave_repl_offset:29397077805
Master 1:
master_host:127.0.0.1
master_port:8883
master_link_status:down
master_last_io_seconds_ago:-1
master_sync_in_progress:1
slave_repl_offset:1
master_sync_left_bytes:-1065039834
master_sync_last_io_seconds_ago:19
master_link_down_since_seconds:1585907337
Master 2:
master_host:127.0.0.1
master_port:8884
master_link_status:up
master_last_io_seconds_ago:0
master_sync_in_progress:0
slave_repl_offset:30033348138
slave_priority:100
slave_read_only:0
connected_slaves:3
slave0:ip=127.0.0.1,port=8883,state=online,offset=5459906740,lag=1
slave1:ip=127.0.0.1,port=8882,state=online,offset=5479231039,lag=0
slave2:ip=127.0.0.1,port=8884,state=online,offset=5484600772,lag=0
master_replid:40182ae939e68fb24b624e0d4e5b38e6004ac9df
master_replid2:0000000000000000000000000000000000000000
master_repl_offset:5484600772
second_repl_offset:-1
repl_backlog_active:1
repl_backlog_size:1048576
repl_backlog_first_byte_offset:5483552197
repl_backlog_histlen:1048576
```

**Comment by oliver006** [comment]
Thanks for raising this, wasn't aware of master-master replication in KeyDB and that this needs to be handled specially in redis_exporter.
I personally don't use KeyDB and don't have time currently but happy to review and merge a PR.

**Comment by oliver006** [comment]
Closed by #374

**Comment by oliver006** [comment]
Released as https://github.com/oliver006/redis_exporter/releases/tag/v1.5.3
